### DISCUSSION TITLE
Is there a way to download content from a certain date?

### DISCUSSION THREAD

**Discussion by BrokenInds** [DIGITALCRIMINALS/OnlyFans#360] [General]
I mean the title says it all.
Like, I'm just curious to know if it's possible to download the content on someone's page from today to a month ago.

Thank you.

**Comment by felixtheant** [comment]
I was just about to ask the same thing, interested to know what the answer is.

**Comment by DIGITALCRIMINAL** [comment]
Not yet

**Comment by GreyBeardy** [comment]
if you have the metadata from a scrape you did maybe 2 months ago - then a new scrape would skip the old content if I understand correctly

**Reply by felixtheant** [reply]
This is true but it would be worth having for new scrapes where you wouldn't have any metadata for.
